draft-ietf-quic-transport.md

```diff
@@ -2860,12 +2860,13 @@ valid frames? -->
 ### Sending ACK Frames
 
 An endpoint MUST NOT send more than one packet containing only an ACK frame per
-received packet that contains frames other than ACK and PADDING frames.
-An endpoint MUST NOT send a packet containing only an ACK frame in response
-to a packet containing only ACK or PADDING frames, even if there are packet
-gaps which precede the received packet. This prevents an indefinite feedback
-loop of ACKs. The endpoint MUST however acknowledge packets containing only
-ACK or PADDING frames when sending ACK frames in response to other packets.
+received non-ACK-eliciting packet(ie: one containing only ACK and/or PADDING
```

Comment:
Maybe "MUST NOT send more than one non-ACK-eliciting packet in response to one non-ACK-eliciting packet"?

The packet the endpoint sends can also be a combination of ACK and PADDING.

Reply:
This could get messy with timeouts and retransmissions. You may want to ensure that peer has ACK but there might not be space in a packet being retransmitted.

Reply:
This text is actually backwards, fixing it to remove the non